release notes/v0.58.0.md

```diff
@@ -0,0 +1,7 @@
+k6 `v0.58.0`
```

Comment:
```suggestion
```
Based on how it renders here https://github.com/grafana/k6/releases, I think we can drop it because it sounds to me like a repetition.

Reply:
Remove the next line as well then :P